Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -593,6 +595,8 @@ if (Platform.OS === 'android') {
   );
 } else if (Platform.OS === 'ios') {
   var RCTScrollView = requireNativeComponent('RCTScrollView', ScrollView);
+} else if (Platform.OS === 'windows') {
```

Comment:
No, I want to keep the Windows specific stuff very explicitly isolated.  We may change the behavior later.
________________________________
From: Matthew Podwysocki<mailto:notifications@github.com>
Sent: ‎3/‎15/‎2016 5:08 PM
To: CatalystCode/react-native<mailto:react-native@noreply.github.com>
Cc: Eric Rozell<mailto:erozell@outlook.com>
Subject: Re: [react-native] fix(RCTScrollView): Fix vertical scrolling behavior for RCTScrollView (#263)

> @@ -593,6 +595,8 @@ if (Platform.OS === 'android') {
>    );
>  } else if (Platform.OS === 'ios') {
>    var RCTScrollView = requireNativeComponent('RCTScrollView', ScrollView);
> +} else if (Platform.OS === 'windows') {

Do we want to collapse to a singe if statement here for iOS and Windows?

---
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub:
https://github.com/CatalystCode/react-native/pull/263/files/6cc7d5110507502c651e320e6f304f9fde1755ef#r56241069